packages/transaction-controller/src/types.ts

```diff
@@ -1119,9 +1119,9 @@ export type TransactionError = {
 export type SecurityAlertResponse = {
   reason: string;
   features?: string[];
-  // TODO: Either fix this lint violation or explain why it's necessary to ignore.
+  // This is API specific hence naming convention is not followed.
   // eslint-disable-next-line @typescript-eslint/naming-convention
-  result_type: string;
```

Comment:
Since `string` encompasses all the `BlockaidResultType` options, should we just leave it as `string` to minimise coupling to Blockaid?

Reply:
Done